DAO.sol

```diff
@@ -657,6 +664,8 @@ contract DAO is DAOInterface, Token, TokenSale {
         if (_recipient == serviceProvider 
             || _recipient == address(rewardAccount) 
             || _recipient == address(this)
+            || (_recipient == address(extraBalance)
```

Comment:
Please write a comment here (on top, not next to it ^_^ ) explaining that the DAO can only receive ether from the extraBalance account only once they have already spent as much as that account's balance is.